src/Files.App/ViewModels/MainPageViewModel.cs

```diff
@@ -144,10 +144,8 @@ public async Task UpdateInstancePropertiesAsync(object navigationArg)
 					var rightTabInfo = await GetSelectedTabInfoAsync(paneArgs.RightPaneNavPathParam);
 					windowTitle = $"{leftTabInfo.tabLocationHeader} | {rightTabInfo.tabLocationHeader}";
 				}
-				else
-				{
+				else if (paneArgs.LeftPaneNavPathParam is not null)
```

Comment:
Sorry, I missed your PR and created another one. But I don't think that a null argument to GetSelectedTabInfoAsync will cause an exception.

Reply:
Let's go with your PR